functions/public/Invoke-WPFUIElements.ps1

```diff
@@ -0,0 +1,315 @@
+    $targetGrid.Children.Clear() | Out-Null
+
+    # Add ColumnDefinitions to the target Grid
+    for ($i = 0; $i -lt $columncount; $i++) {
+        $colDef = New-Object Windows.Controls.ColumnDefinition
+        $colDef.Width = New-Object Windows.GridLength(1, [Windows.GridUnitType]::Star)
+        $targetGrid.ColumnDefinitions.Add($colDef) | Out-Null
+    }
+
+    # Only apply the logic for distributing entries across columns if the targetGridName is "appspanel"
+    if ($targetGridName -eq "appspanel") {
+        $panelcount = 0
+        $entrycount = $configHashtable.Keys.Count + $organizedData["0"].Keys.Count
+        $maxcount = [Math]::Round($entrycount / $columncount + 0.5)
+    }
+
+    # Iterate through 'organizedData' by panel, category, and application
+    $count = 0
+    foreach ($panelKey in ($organizedData.Keys | Sort-Object)) {
+        # Create a Border for each column
+        $border = New-Object Windows.Controls.Border
+        $border.VerticalAlignment = "Stretch" # Ensure the border stretches vertically
+        [System.Windows.Controls.Grid]::SetColumn($border, $panelcount)
+        $border.style = $window.FindResource("BorderStyle")
+        $targetGrid.Children.Add($border) | Out-Null
+
+        # Create a StackPanel inside the Border
+        $stackPanel = New-Object Windows.Controls.StackPanel
+        $stackPanel.Background = [Windows.Media.Brushes]::Transparent
+        $stackPanel.SnapsToDevicePixels = $true
+        $stackPanel.VerticalAlignment = "Stretch" # Ensure the stack panel stretches vertically
+        $border.Child = $stackPanel
+        $panelcount++
+
+        foreach ($category in ($organizedData[$panelKey].Keys | Sort-Object)) {
+            $count++
+            if ($targetGridName -eq "appspanel" -and $columncount -gt 0) {
+                $panelcount2 = [Int](($count) / $maxcount - 0.5)
+                if ($panelcount -eq $panelcount2) {
+                    # Create a new Border for the new column
+                    $border = New-Object Windows.Controls.Border
+                    $border.VerticalAlignment = "Stretch" # Ensure the border stretches vertically
+                    [System.Windows.Controls.Grid]::SetColumn($border, $panelcount)
+                    $border.style = $window.FindResource("BorderStyle")
+                    $targetGrid.Children.Add($border) | Out-Null
+
+                    # Create a new StackPanel inside the Border
+                    $stackPanel = New-Object Windows.Controls.StackPanel
+                    $stackPanel.Background = [Windows.Media.Brushes]::Transparent
+                    $stackPanel.SnapsToDevicePixels = $true
+                    $stackPanel.VerticalAlignment = "Stretch" # Ensure the stack panel stretches vertically
+                    $border.Child = $stackPanel
+                    $panelcount++
+                }
+            }
+
+            $label = New-Object Windows.Controls.Label
+            $label.Content = $category -replace ".*__"
```

Comment:
```suggestion
            $label.Content = $category -replace ".*__", ""
```
Sticking with to a clear style of using a method or operator is better, not to mention using the suggested snippet of code is clearer to the reader to what this line do, it sets the value of `$label.Content` to be the same as `$category`'s value.. **without** `ANY-CHARS__` prefix.